app/views/home/index.html.erb

```diff
@@ -8,9 +8,7 @@
               <img src="/assets/panel-image-1.png" alt="" />
             </a>
           </div>
-
           <div class="col-md-5">
-
           <% if user_signed_in? %>
```

Comment:
Yes I dont really understand the indentation here either, I'll make some changes and then I'll push it all up again :)